src/Core/AdminConsole/Entities/Organization.cs

```diff
@@ -94,6 +94,9 @@ public class Organization : ITableObject<Guid>, IStorableSubscriber, IRevisable,
     /// they have Can Manage permissions for.
     /// </summary>
     public bool LimitCollectionCreationDeletion { get; set; }
+    public bool LimitCollectionCreation { get => LimitCollectionCreationDeletion; set => LimitCollectionCreationDeletion = value; }
+    public bool LimitCollectionDeletion { get => LimitCollectionCreationDeletion; set => LimitCollectionCreationDeletion = value; }
```

Comment:
I don't think this is necessary because it's handled at the db layer. The values are `COALESCE`d with `LimitCollectionCreationDeletion` on both read and write operations, and I think that's best to do in one place only.

Reply:
They are `COALESCE`d in the Dapper implementation, but without the use of stored procedures in the Entity Framework implementation we need to keep the values in sync some other way. This seemed like the least intrusive way to do that, to me.

Reply:
This comment made me realise - usually I do my MSSQL migrations 1 release before the code changes, but I do the EF migrations at the same time as the code changes, given that it's code-first and doesn't have to support EDD. That would mean there are no changes to the C# code at all (including this class), just silently adding the MSSQL column and keeping it in sync.

I think this approach should also work but I haven't tried it myself. As long as EF uses the getters and setters properly (no tricky reflection) then I agree it should stay in sync.

Reply:
That makes sense. I did the same thing at first, but decided to try this instead to keep the database types as 1:1 as possible across releases. We can cut it out and just do the SQL here if that's anybody else's preference.

Reply:
Nope, I think your approach here is a valid one as well!